### PR TITLE
cardano-node:  factor tracing-independent mini-API as Cardano.Node.Queries

### PR DESCRIPTION
**Pull request by deepfire**
The current tracing system makes a number of projections of the system state, which are not really tracing-system specific.

We consolidate those as modules under `Cardano.Node.Queries`, so this can be reused by old and new tracing systems:

  - Cardano.Node.Queries.ConvertTxId
  - Cardano.Node.Queries.KES
  - Cardano.Node.Queries.Ledger

### CODE REVIEW

**Review by newhoggy — APPROVED** [review]
LGTM

File: cardano-node/src/Cardano/Node/Queries/KES.hs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Cardano.Node.Queries.KES
+  (
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Node/Queries/KES.hs:12]: Extra newline here

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
Rather than creating three separate modules, lets put all of this into one module.

File: cardano-node/cardano-node.cabal [hunk 2]
```diff
@@ -73,16 +73,15 @@ library
                        Cardano.Node.Protocol.Shelley
                        Cardano.Node.Protocol.Types
                        Cardano.Node.Parsers
+                       Cardano.Node.Queries
```
Comment (Jimbo4350) [inline cardano-node/cardano-node.cabal:76]: Thanks :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Pull request merged**